internal/integration/integration_test.go

```diff
@@ -378,6 +378,41 @@ func (suite *SpokesReceivePackTestSuite) TestSpokesReceivePackWrongObjectSucceed
 	})
 }
 
+
+func (suite *SpokesReceivePackTestSuite) TestSpokesReceivePackIgnoreArgsSucceed() {
+	assert.NoError(suite.T(), chdir(suite.T(), suite.remoteRepo), "unable to chdir into our remote Git repo")
+	// Disable the `receive.fsckObjects option
```

Comment:
NIT: you either close the starting ` or remove it

Reply:
And the comment shouldn't be the opposite? You are not disabling the fsckObjects option, but enabling it? I don't think you are testing the right thing here

Reply:
Yes you are correct. This is a left over comment from a test I copied from above and forgot to remove. Good catch.